packages/app/src/systems/Bridge/pages/Bridge.tsx

```diff
@@ -52,14 +50,14 @@ export const Bridge = () => {
                 <InputAmount
                   balance={assetBalance}
                   asset={{
```

Comment:
can we pass in units here as well?

Reply:
we should not. units will be always fuel side.

imagine if the user bridges from eth more units than fuel supports, what will happen with remaining value?

parsing to units of ETH will happen in bridge orchestrator service